tests/all/js_globals/Symbol.rs

```diff
@@ -0,0 +1,37 @@
+#![allow(non_snake_case)]
+
+use project;
+
+#[test]
+fn has_instance() {
+    project()
+        .file(
+            "src/lib.rs",
+            r#"
+            #![feature(proc_macro, wasm_custom_section)]
+
+            extern crate wasm_bindgen;
+            use wasm_bindgen::prelude::*;
+            use wasm_bindgen::js;
+
+            #[wasm_bindgen]
+            pub fn symbol_has_instance() -> js::Symbol {
+                js::Symbol::has_instance()
+            }
+
+        "#,
+        )
+        .file(
+            "test.ts",
+            r#"
+            import * as assert from "assert";
+            import * as wasm from "./out";
+
+            export function test() {
+                assert.ok(wasm.symbol_has_instance());
+                assert.ok([] instanceof Array);
```

Comment:
Correct me if I'm wrong, but I think that line alone doesn't bring much value in this test. Could you instead try to include a code snippet from [Symbol.hasInstance page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/hasInstance), asserting that `[] instanceof Array1` (notice that `Array1` is a new class here, which implements `static [Symbol.hasInstance](instance)` method).

Reply:
Oh, I missed to revert chaging to create test that absolutely passing :bowing_man:

Reply:
👍 although now that I'm thinking about it again, it might be even pointless to include that code snippet - that would be testing the JS itself (not the WASM part).